samples/client/petstore/elixir/test/deserializer_test.exs

```diff
@@ -25,25 +24,28 @@ defmodule DeserializerTest do
         "name": "sea"
       }
     ],
-    "status": "foo"
+    "status": "available"
```

Comment:
The previous version did not validate enum values. This is one of the nice goodies we get from switching to Ecto.